docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc

```diff
@@ -45,6 +73,36 @@ refer to <<ml-nlp-overview>>.
 * https://huggingface.co/elastic/distilbert-base-cased-finetuned-conll03-english
 * https://huggingface.co/philschmid/distilroberta-base-ner-conll2003
 
+[discrete]
+=== Named entity recognition expected model output
+
+Named entity recognition is a specific token classification task. Each token in the sequence is scored related to 
+a specific set of classification labels. For the Elastic Stack, we use Inside-Outside-Beginning (IOB) tagging. Additionally,
+ only the following classification labels are supported: "o", "b_misc", "i_misc", "b_per", "i_per", "b_org", "i_org", "b_loc", "i_loc".
+
+The `"o"` entity label indicates that the current token is outside any entity.
+`"b"` indicates the beginning of an entity.
+`"misc"` is a miscellaneous entity.
+`"loc"` is a location.
+`"per"` is a person.
+`"org"` is an organization.
+
+The response format of the model must be <number of sequences inferred> by <number of tokens in sequence> by <classification score for each entity for that token>.
```

Comment:
> The response format of the model must be <number of sequences inferred> by <number of tokens in sequence> by <classification score for each entity for that token>.

Per my suggestion for the previous section, I suggest using something like this instead: "It is a nested set of {arrays? objects? tuples?} for each inferred sequence, where every token in every sequence has classification scores for each entity label."